app/views/contentful/static/show.html.slim

```diff
@@ -0,0 +1,9 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds-from-desktop
+
+    h1= @model.heading
+    p= @model.subheading
```

Comment:
Maybe only have the subheading display if it exists (so just put inside the conditional) - otherwise you have empty <p></p>